packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -265,29 +283,45 @@ export const prepareLocalOrchestrationAccountKit = (
        */
       returnVoidWatcher: {
         /**
-         * @param {Record<unknown, unknown>[]} results
+         * @param {any} _result
          */
-        onFulfilled(results) {
-          results.length === 1 ||
-            Fail`expected exactly one result; got ${results}`;
-          trace('Result', results[0]);
+        onFulfilled(_result) {
           return undefined;
         },
       },
+      /**
+       * handles a request for balance from a bank purse and returns the balance
+       * as a Chain Amount
```

Comment:
I was going to suggest writing "returns the balance as a Chain Amount" using JSDoc so the computer can help us maintain it... I see you've got `@returns {DenomAmount}`, so in some sense you already did it, but in fact this proves the point: it should be `DenomAmount`, not "Chain Amount".

I'm not sure this comment is worth the bother. It doesn't actually handle a request for a balance. It provides callbacks for a watch and such.